parallel-consumer-core/src/main/java/io/confluent/parallelconsumer/state/PartitionState.java

```diff
@@ -629,5 +631,30 @@ boolean checkIfWorkIsStale(final WorkContainer<?, ?> workContainer) {
         return false;
     }
 
-}
+    public PCMetrics.PCPartitionMetrics getMetrics() {
```

Comment:
Should we invert this? Move the construction into the metrics class, instead of the state class? (pass the metrics class the state)